Assets/Script/Backend/GameStages/GameSceneTest.cs

```diff
@@ -4,8 +4,8 @@ public class GameSceneTest : GameSceneBase
     {
         public override void InitializeCharacters()
         {
-            var enemyName = "EnemyDefault";
-            AddGameObject(new Enemy(enemyName, LoadEnemyStats(enemyName), LoadDefaultEnvironmentalStats(isEnemy: true)));
+            //var enemyName = "EnemyDefault";
```

Comment:
Comment or remove?

Reply:
Also, this is so that we ever need it again in the future, if we don't need it, we can remove it at any time